### PR TITLE
ARROW-7470: [JS] fix typos

### PR DESCRIPTION
**Pull request by kiszk**
This PR fixes typos in files under `js` directory

### CODE REVIEW

File: js/gulp/package-task.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const createMainPackageJson = (target, format) => (orig) => ({
     module: `${mainExport}.dom.mjs`,
     types: `${mainExport}.node.d.ts`,
     unpkg: `${mainExport}.es5.min.js`,
-    jsdelivr: `${mainExport}.es5.min.js`,
+    jsdeliver: `${mainExport}.es5.min.js`,
```
Comment (wesm) [inline js/gulp/package-task.js]: I don't think this is a typo

https://www.jsdelivr.com/

**Review by wesm — COMMENTED** [review]
Aside from my comment about jsdelivr this looks ok, thank you =)

File: js/gulp/package-task.js (hunk 1 above)
Comment (kiszk) [reply js/gulp/package-task.js]: Thank you for pointing it out. I have just reverted the changes.

**Review by kou — APPROVED** [review]
+1

@kiszk You can assign yourself as "Assignee" on JIRA issue when you will open a pull request. And you don't need to set not released version ("1.0.0" for this case) to "Affect Version/s" on JIRA issue.

**Comment by kiszk** [comment]
@kou Thank you for your suggestions. I will do them next.

**Pull request closed without merging**